fault-tolerance/src/main/java/io/helidon/faulttolerance/Timeout.java

```diff
@@ -120,6 +121,50 @@ public Builder cancelSource(boolean cancelSource) {
             return this;
         }
 
+        /**
+         * <p>
+         * Load all properties for this circuit breaker from configuration.
+         * </p>
+         * <table class="config">
+         * <caption>Configuration</caption>
+         * <tr>
+         *     <th>key</th>
+         *     <th>default value</th>
+         *     <th>description</th>
+         * </tr>
+         * <tr>
+         *     <td>timeout</td>
+         *     <td>10 seconds</td>
+         *     <td>Length of timeout</td>
+         * </tr>
+         * <tr>
+         *     <td>current-thread</td>
+         *     <td>false</td>
+         *     <td>Control that task is executed in calling thread</td>
+         * </tr>
+         * <tr>
+         *     <td>name</td>
+         *     <td>Timeout-N</td>
+         *     <td>Name used for debugging</td>
+         * </tr>
+         * <tr>
+         *     <td>cancel-source</td>
+         *     <td>true</td>
+         *     <td>Cancel task source if task is cancelled</td>
+         * </tr>
+         * </table>
+         *
+         * @param config the config node to use
+         * @return updated builder instance
+         */
+        public Builder config(Config config) {
```

Comment:
Now that config is supported, we should also add the configuration metadata annotations + annotation processor